src/absil/ilwrite.fs

```diff
@@ -1413,10 +1413,17 @@ let rec GetCustomAttrDataAsBlobIdx cenv (data:byte[]) =
 
 and GetCustomAttrRow cenv hca attr = 
     let cat = GetMethodRefAsCustomAttribType cenv attr.Method.MethodRef
-    UnsharedRow 
-        [| HasCustomAttribute (fst hca, snd hca)
-           CustomAttributeType (fst cat, snd cat) 
-           Blob (GetCustomAttrDataAsBlobIdx cenv attr.Data) |]  
+    for element in attr.Elements do
+        match element with
+        | ILAttribElem.Type (Some ty) -> GetTypeRefAsTypeRefIdx cenv ty.TypeRef |> ignore
```

Comment:
The ``ty.TypeRef`` is failing here.  You might need to check when ``ILAttribElem.Type `` occurs